lib/plug/ssl.ex

```diff
@@ -197,6 +282,51 @@ defmodule Plug.SSL do
     |> Keyword.put_new(:reuse_sessions, true)
   end
 
+  defp configure_managed_tls(options) do
+    {cipher_suite, options} = Keyword.pop(options, :cipher_suite)
+
+    case cipher_suite do
+      :strong -> set_strong_tls_defaults(options)
+      :compatible -> set_compatible_tls_defaults(options)
+      _ -> options
+    end
+  end
+
+  defp set_managed_tls_defaults(options) do
+    options
+    |> Keyword.put_new(:honor_cipher_order, true)
+    |> Keyword.put_new(:client_renegotiation, false)
+    |> Keyword.put_new(:eccs, @eccs)
+  end
+
+  defp set_strong_tls_defaults(options) do
+    options
+    |> set_managed_tls_defaults
+    |> Keyword.put_new(:ciphers, @strong_tls_ciphers)
+    |> Keyword.put_new(:versions, [:"tlsv1.2"])
+  end
+
+  defp set_compatible_tls_defaults(options) do
+    options
+    |> set_managed_tls_defaults
+    |> Keyword.put_new(:ciphers, @compatible_tls_ciphers)
+    |> Keyword.put_new(:versions, [:"tlsv1.2", :"tlsv1.1", :tlsv1])
+  end
+
+  defp validate_ciphers(options) do
+    Keyword.get(options, :ciphers, [])
+    |> Enum.map(&validate_cipher/1)
+
+    options
+  end
+
+  defp validate_cipher(cipher) do
+    if is_binary(cipher) do
+      message = "Your cipher list contained the binary \"#{cipher}\", which is not valid"
```

Comment:
All of the other error messages start in lowercase, so let's do this here too. I also think we can tell users what they should do right, something like:

 > invalid cipher #{inspect(cipher)} in cipher list. Strings (double-quoted) are not allowed in ciphers. Ciphers must be either charlists (single-quoted) or tuples. See the ssl application docs for reference

Reply:
Done.